lib/PuppeteerSharp.Tests/ElementHandleTests/ClickTests.cs

```diff
@@ -25,7 +25,7 @@ public async Task ShouldWork()
         public async Task ShouldWorkForShadowDomV1()
         {
             await Page.GoToAsync(TestConstants.ServerUrl + "/shadow.html");
-            var buttonHandle = (await Page.EvaluateExpressionHandleAsync("button")) as ElementHandle;
+            var buttonHandle = (ElementHandle) await Page.EvaluateExpressionHandleAsync("button");
```

Comment:
no null check after an `as` implied it can only be that type. hence a cast is better, since if the assumption is incorrect you will get a useful cast exception instead of a useless null ref exception